src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.csproj.FileListAbsolute.txt

```diff
@@ -0,0 +1,15 @@
+C:\GitRepo\epr-calculator-api\src\EPR.Calculator.API.Data\bin\Debug\net8.0\EPR.Calculator.API.Data.deps.json
```

Comment:
The obj directory can be added to .gitignore.

Reply:
Done